portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,32 +33,53 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private static final String[] COMMENTS = {
-    "Wow good job on the first portfolio website",
-    "You should find some help, the design is wacky",
-    "Where am I?"
-  };
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String json = convertArrayToJson(COMMENTS);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
move this out of doGet and make it `private static final`

Reply:
done